live-examples/css-examples/images/function-repeating-conic-gradient.html

```diff
@@ -0,0 +1,44 @@
+<section id="example-choice-list" class="example-choice-list large">
+
+<div class="example-choice">
+<pre><code class="language-css">background-image: repeating-conic-gradient(red, orange, yellow, green, blue 60deg);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background-image: repeating-conic-gradient(
+     from 0.25turn at 50% 30%, 
+     blue 0 0.1turn, lightblue 0.1turn 0.2turn);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background-image: repeating-conic-gradient(
+    from 3deg at 25% 25%, 
```

Comment:
Unless I'm missing something, it might be nice to omit `from 3deg ` here, to show that you can omit this in the syntax.